### PR TITLE
Add x/incentives cli docs

### PR DESCRIPTION
**Pull request by mconcat**

### CODE REVIEW

File: x/incentives/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+The incentives module provides users the functionality to create gauges, which
+distributes reward tokens to the qualified lockups. Each lockup has designated
+lockup duration that indicates how much time that the user have to wait until
+the token release after they request to unlock the tokens. 
+
+## Creating Gauges
+
+To initialize a gauge, the creator should decide the following parameters:
+- Distribution condition: qualified denom and minimum lockup duration.
```
Comment (ValarDragon) [inline x/incentives/README.md:11]:
```suggestion
- Distribution condition: denom to incentivize and minimum lockup duration.
```

File: x/incentives/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- Rewards: tokens to be distributed to the lockup owners.
+- Start time: time when the distribution will begin.
+- Total epochs: period of distributions in epochs.
```
Comment (ValarDragon) [inline x/incentives/README.md:14]:
```suggestion
- Total epochs: number of epochs to distribute over. (Osmosis epochs are 1 day each, ending at 5PM UTC everyday)
```

File: x/incentives/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+```
+I want to make incentives for LP tokens of pool X, namely LPToken, that have been locked up for at least 1 day.
+I want to reward 1000 Mytoken to this pool over 2 weeks (2 epochs). (500 rewarded on each day)
+I want the rewards to start disbursing at 2022 Jan 01.
+
+MsgCreateGauge:
+- Distribution condition: denom "LPToken", 1 day.
+- Rewards: 1000 MyToken
+- Start time: 2022-01-01T00:00:00Z (in RFC3339 format)
+- Total epochs: 2 (weeks)
```
Comment (ValarDragon) [inline x/incentives/README.md:40]:
```suggestion
- Total epochs: 2 (days)
```

File: x/incentives/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  --epochs-duration [total distribution duration]
+  --perpetual
+```
+
+### Examples
+
+#### Case 1
+```
+I want to make incentives for LP tokens of pool X, namely LPToken, that have been locked up for at least 1 day.
+I want to reward 1000 Mytoken to this pool over 2 weeks (2 epochs). (500 rewarded on each day)
```
Comment (ValarDragon) [inline x/incentives/README.md:33]:
```suggestion
I want to reward 1000 Mytoken to this pool over 2 days (2 epochs). (500 rewarded on each day)
```

File: x/incentives/README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+osmosisd tx incentives create-gauge LPToken 1000MyToken \
+  --duration 24h \
+  --start-time 2022-01-01T00:00:00Z \
+  --epochs 2 # or --epochs-duration 336h
+```
+
+#### Case 2
+
+I want to make incentives for atoms that have been locked up for at least 1 month.
+I want to reward 1000 MyToken to atom holders perpetually.
```
Comment (ValarDragon) [inline x/incentives/README.md:52]:
```suggestion
I want to reward 1000 MyToken to atom holders perpetually. (Meaning I add more tokens to this gauge myself every epoch)
```

File: x/incentives/client/cli/flags.go [hunk 6]
```diff
@@ -1,27 +1,32 @@
...
 
 // FlagSetCreateGauge returns flags for creating gauge
 func FlagSetCreateGauge() *flag.FlagSet {
 	fs := flag.NewFlagSet("", flag.ContinueOnError)
 
-	fs.String(FlagLockQueryType, "ByDuration", "ByDuration | ByTime")
-	fs.String(FlagDenom, "stake", "locked denom to be queried")
-	fs.String(FlagDuration, "168h", "The duration token to be locked, default 1w(168h). Other examples are 1h, 1m, 1s, 0.1s. Maximum unit is hour.")
-	fs.Int64(FlagTimestamp, 1615917475, "Timestamp to that started tokens lock")
+	dur, _ := time.ParseDuration("168h")
```
Comment (ValarDragon) [inline x/incentives/client/cli/flags.go:26]:
```suggestion
	dur, _ := time.ParseDuration("24h")
```

File: x/incentives/client/cli/flags.go [hunk 7]
```diff
@@ -1,27 +1,32 @@
...
+	fs.Duration(FlagDuration, dur, "The duration token to be locked, default 1w(168h). Other examples are 1h, 1m, 1s, 0.1s. Maximum unit is hour.")
```
Comment (ValarDragon) [inline x/incentives/client/cli/flags.go:27]:
```suggestion
	fs.Duration(FlagDuration, dur, "The duration token to be locked, default 1d(24h). Other examples are 7d(168h), 14d(336h). Maximum unit is hour.")
```

File: x/incentives/client/cli/tx.go [hunk 8]
```diff
@@ -36,77 +36,71 @@ func GetTxCmd() *cobra.Command {
 // NewCreateGaugeCmd broadcast MsgCreateGauge
 func NewCreateGaugeCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create-gauge [coins] [start_time] [num_epochs_paid_over] [flags]",
+		Use:   "create-gauge [denom] [reward] [flags]",
```
Comment (ValarDragon) [inline x/incentives/client/cli/tx.go:39]:
```suggestion
		Use:   "create-gauge [lockup_denom] [reward] [flags]",
```

File: x/incentives/README.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+#### Case 1
+```
+I want to make incentives for LP tokens of pool X, namely LPToken, that have been locked up for at least 1 day.
+I want to reward 1000 Mytoken to this pool over 2 weeks (2 epochs). (500 rewarded on each day)
+I want the rewards to start disbursing at 2022 Jan 01.
+
+MsgCreateGauge:
+- Distribution condition: denom "LPToken", 1 day.
+- Rewards: 1000 MyToken
+- Start time: 2022-01-01T00:00:00Z (in RFC3339 format)
```
Comment (ValarDragon) [inline x/incentives/README.md:38]: Lets change this to unix time, imo will be easier

File: x/incentives/README.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+MsgCreateGauge:
+- Distribution condition: denom "atom", 720 hours.
+- Rewards: 1000 MyTokens
+- Start time: empty(immedietly)
+- Total epochs: 1 (perpetual)
+
+```bash
+osmosisd tx incentives create-gauge atom 1000MyToken
+  --perpetual \  
+  --duration 720h
```
Comment (ValarDragon) [inline x/incentives/README.md:63]:
```suggestion
  --duration 168h 
```

**Comment by ValarDragon** [comment]
Sunny reviewed as well
